README.md

```diff
@@ -32,13 +32,18 @@ Q2: What is the answer?
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+_Lydia_:: I wanted to move to a developer role. My degree has some programming courses to. So wanted to return to my roots kinda ...
 
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+_Lydia_:I love to travel as well. My favorite places so far has been Puerto Rico. I always love travelling back to India.
```

Comment:
Puerto Rico is _so_ beautiful!  I was only there a few days where my cruise disembarked, but it was so nice!

Reply:
I agree, it is beautiful. Which part of Puerto Rico did your cruise ship stop at. I loved the Flamenco and El Yunque rain forest.